libblkid/src/superblocks/ext.c

```diff
@@ -190,6 +190,11 @@ static void ext_get_info(blkid_probe pr, int ver, struct ext2_super_block *es)
 
 	if (le32_to_cpu(es->s_log_block_size) < 32)
 		blkid_probe_set_block_size(pr, 1024U << le32_to_cpu(es->s_log_block_size));
+
+	uint64_t fslastblock = es->s_blocks_count |
+		((es->s_feature_incompat & EXT4_FEATURE_INCOMPAT_64BIT) ?
+		(uint64_t) es->s_blocks_count_hi << 32 : 0);
```

Comment:
I guess you don't want access `es->` stuff without leXX_to_cpu(), it means
```
   le32_to_cpu(es->s_feature_incompat) 
   le32_to_cpu(es->s_blocks_count_hi)
```
Note that es->s_feature_incompat is used on more places in the function, maybe we can create a local variable with it rather than use  le32_to_cpu(es->s_feature_incompat) on all places,

Reply:
oh, of course, not enough coffee :D